modules/services/cloudwatch/ec2-monitoring.tf

```diff
@@ -0,0 +1,22 @@
+## Default EC2 monitoring metrics
+
+resource "aws_cloudwatch_metric_alarm" "high_cpu_alert_ec2" {
+  count             = length(var.ec2_instances)
+  alarm_name        = "High CPU Utilization - EC2 Instance ${element(var.ec2_instances, count.index)}-${var.customer_name}-${var.environment}"
+  alarm_description = "CPU utilization exceeded acceptable threshold"
+
+  actions_enabled = var.default_ec2_alarms_enabled
+  alarm_actions   = [aws_sns_topic.high_priority_alert.arn]
+  ok_actions      = [aws_sns_topic.high_priority_alert.arn]
+
+  comparison_operator = "GreaterThanOrEqualToThreshold"
+  evaluation_periods  = "12"
+  threshold           = "90"
+  metric_name         = "CPUUtilization"
+  statistic           = "Average"
+  period              = "300"
```

Comment:
Here is something to think about, when we say period = 300 seconds and 12 evaluation period it translates to if the CPU utilization is above 90% for 12 continuos time over a period of 12*5(300) mins = 1 hour.

Hence if the CPU utilization is above 90% for 1 hour it will raise an alarm. I feel this is like a long shot.

We can do something like `period=120` and evaluation_period=3, so it roughly translated to CPU utilization being > 90 for 6 mins.

Reply:
I agree with you here .. 1 hour is indeed too long for the cpu utilization to be so high before raising alarm.

I have reduced the `period` to 60 and kept the `evaluation_period` to 6, so that it is evaluated every 1 minute and the alarm would trip at 6th minute